src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -798,31 +798,31 @@ Public Module VerificationHelpers
         Else
             Dim prev As SyntaxTrivia? = Nothing
             For Each tr In node.AsToken.LeadingTrivia
-                If tr.HasStructure Then
-                    InternalVerifyNoAdjacentTriviaHaveSameKind(tr.GetStructure)
-                End If
-                If prev IsNot Nothing Then
-                    Assert.True(prev.Value.Kind <> tr.Kind,
-                                "Both current and previous trivia have Kind=" & tr.Kind.ToString &
-                                " [See under TokenKind=" & node.Kind().ToString & ", NonTerminalKind=" & node.Parent.Kind.ToString & "]")
-                End If
+                VerifyNoAdjacentTriviaHaveSameKind(node, tr, prev)
                 prev = tr
             Next
             prev = Nothing
             For Each tr In node.AsToken.LeadingTrivia
-                If tr.HasStructure Then
-                    InternalVerifyNoAdjacentTriviaHaveSameKind(tr.GetStructure)
-                End If
-                If prev IsNot Nothing Then
-                    Assert.True(prev.Value.Kind <> tr.Kind,
-                                "Both current and previous trivia have Kind=" & tr.Kind.ToString &
-                                " [See under TokenKind=" & node.Kind().ToString & ", NonTerminalKind=" & node.Parent.Kind.ToString & "]")
-                End If
+                VerifyNoAdjacentTriviaHaveSameKind(node, tr, prev)
                 prev = tr
             Next
```

Comment:
I think there is a bug in the second loop, it should probably go over TrailingTrivia. I'll fix that.